viadot/sources/genesys.py

```diff
@@ -0,0 +1,516 @@
+import json
+import base64
+import warnings
+import asyncio
+from typing import Any, Dict, List, Literal
+
+import prefect
+import aiohttp
+import pandas as pd
+from datetime import datetime, timedelta
+from aiolimiter import AsyncLimiter
+
+from viadot.config import local_config
+from viadot.sources.base import Source
+from ..exceptions import CredentialError, APIError
```

Comment:
use the same structure for all import i.e viadot.exceptions instead ..exceptions